components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8715,6 +8744,69 @@ def setChannelInverted(self, channelIndex, inverted=True):
         if inverted:
             self._re.addCodomainMapToChannel(r, channelIndex)
 
+    @assert_re()
+    def getFamilies(self):
+        """
+        Gets a dict of available families.
+
+        :return:    Families
+        :rtype:     Dict
+        """
+        if not len(self._qf):
+            for f in [BlitzObjectWrapper(self._conn, f)
+                      for f in self._re.getAvailableFamilies()]:
+                self._qf[f.value.lower()] = f
+        return self._qf
+
+    @assert_re()
+    def setQuantizationMap(self, channelIndex, family, coefficient):
+        """
+        Sets the quantization strategy to the given family
+        and coefficient
+
+        :param channelIndex:    The index of channel (int)
+        :param family:          The family (string)
+        :param coefficient:     The coefficient (float)
+        """
+        if channelIndex < 0 or channelIndex >= self.getSizeC():
+            return
+
+        f = self.getFamilies().get("linear")
+        try:
+            f = self._qf.get(family.lower(), f)
+        except:
+            pass
```

Comment:
Minor point, how about
```
families = self.getFamilies()
f = families.get("linear")
try:
    f = families.get(family.lower(), f)
...
```
Getting the same dict of families in 2 different ways is slightly more confusing, and could lead to errors if someone were to try accessing ```self._qf``` without first calling ```self.getFamilies()```.